src/blockchain_db/blockchain_db.h

```diff
@@ -1884,7 +1884,7 @@ class BlockchainDB
   // TODO: description
   virtual uint64_t get_path_by_global_output_id(const std::vector<uint64_t> &global_output_ids,
     const uint64_t as_of_n_blocks,
-    std::vector<uint64_t> &leaf_idxs_out,
+    std::vector<fcmp_pp::curve_trees::AssignedLeafIdx> &leaf_idxs_out,
```

Comment:
Sorry I'm slightly confused why this was changed? Aren't all of them going to be assigned or else the whole thing fails?